drivers/xen/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 
 zephyr_sources(hvm.c)
 zephyr_sources(events.c)
+zephyr_sources(memory.c)
```

Comment:
One comment: I believe this was added by mistake, it is already present on line #8